assets/js/langpack/es.json

```diff
@@ -0,0 +1,126 @@
+{
+  "token": {
+    "Claim control of your":
+    "Obtén el control de tus",
+
+    "Docker images":
+    "Imágenes Docker",
+
+    "View on GitHub":
+    "Ver en GitHub",
+
+    "Download .zip":
+    "Descargar .zip",
+
+    "Download .tar.gz":
+    "Descargar .tar.gz",
+
+    "What is Portus":
+    "Qué es Portus",
+
+    "Portus is an open source authorization service and user interface for the next generation Docker Registry.":
+    "Portus es un servicio de autorización de libre código e interfaz de usuario para la próxima generación de Registros Docker",
+
+    "Why use Portus":
+    "Por qué Portus",
+
+    "The Docker Hub is a wonderful place, but what if you cannot, or do not wish to push your images to a thrid party registry? You may setup a private instance of the Docker registry, but then everyone in your organization has push and pull privileges over all of your images.":
+    "El Docker Hub es un lugar maravilloso, pero si no puedes, o no deseas hacer push de tus imágenes a registros de terceros? Puedes configurar una instancia privada del registro de Docker, pero entonces cualquiera en tu organización tiene privilegios para hacer push y pull de todas tus imágenes.",
```

Comment:
Since it's a question: "El Docker Hub es un lugar maravillos pero, y si no puedes o no deseas...". Notice the change in commas and the "y".